Svc/SystemResources/SystemResources.fpp

```diff
@@ -0,0 +1,146 @@
+module Svc {
+
+  enum SystemResourceEnabled {
+    DISABLED = 0
+    ENABLED = 1
+  }
+
+  passive component SystemResources {
+
+    @ Run port
+    guarded input port run: [1] Svc.Sched
+    
+    # ----------------------------------------------------------------------
+    # Special ports
+    # ----------------------------------------------------------------------
+
+    @ Time get port
+    time get port Time
+
+    @ Command registration port
+    command reg port CmdReg
+
+    @ Command received port
+    command recv port CmdDisp
+
+    @ Command response port
+    command resp port CmdStatus
+
+    @ Text event port
+    text event port LogText
+
+    @ Event port
+    event port Log
+
+    @ Telemetry port
+    telemetry port Tlm
+
+
+    @ A command to enable or disable system resource telemetry
+    guarded command ENABLE(
+                            enable: SystemResourceEnabled @< whether or not system resource telemetry is enabled
+                          ) \
+      opcode 0
+
+    @ Report version as EVR
+    guarded command VERSION \
+      opcode 1
+
+    @ Version of the git repository.
+    event VERSION(
+                   version: string size 40 @< version string
+                 ) \
+      severity activity low \
+      id 0 \
+      format "Version: [{}]"
+
+    @ Total system memory in KB
+    telemetry MEMORY_TOTAL: U64 id 0 \
+      format "{} KB"
+
+    @ System memory used in KB
+    telemetry MEMORY_USED: U64 id 1 \
+      format "{} KB"
+
+    @ System non-volatile available in KB
+    telemetry NON_VOLATILE_TOTAL: U64 id 2 \
+      format "{} KB"
+
+    @ System non-volatile available in KB
+    telemetry NON_VOLATILE_FREE: U64 id 3 \
+      format "{} KB"
+
+    @ FPP from XML: could not translate format string "%3.2f percent"
+    @ System's CPU Percentage
+    telemetry CPU: F32 id 4
+
+    @ FPP from XML: could not translate format string "%3.2f percent"
```

Comment:
Some format strings were not translated here. We should update them by hand. It should be something like `format "{.2f} percent"`